README.md

```diff
@@ -71,6 +71,45 @@ Then the following data is inserted in the store:
         as:deleted "thetimeofdelete"^^xsd:dateTime  .
 ```
 
+## Extending Forms
+
+### Predicates and Types
+
+Use the same model as regular forms, with the following differences:
+-   don't use the `sh:group` predicate to denote connections to the owning form (or its sections/listings). Instead use `ext:extendsGroup`
+-   don't use the `form:includes` predicate to denote that a field is part of a form. Instead use `ext:extendsForm`
+
+An extension is represented by 
```

Comment:
Not really clear to me where this part comes from, is this something provided by the forms, do we want to add this ...

Reply:
we are adding this so we can reference/control it from the mu-resources world

Reply:
So we will be adding this to ember-submission-form-fields? or is this something we add in this service which then converts it to a form ember-submission-form-fields can read?

Reply:
indeed our service translates this to regular forms (see transparency)